Documentation/Plus Codes Specification.md

```diff
@@ -0,0 +1,119 @@
+
+Plus Codes are characterized by their **code length**. This specifies the distance between the northern & southern bounds and the western & eastern bounds. Plus codes MUST use one of these code lengths:
+
+| Code length | North-south distance | West-east distance |  Precision   |
+| :---------: | -------------------- | ------------------ | :----------: |
+|      2      | 20 degrees           | 20 degrees         |  < 2300 km   |
+|      4      | 1 degrees            | 1 degrees          |   < 120 km   |
+|      6      | 1/20 degrees         | 1/20 degrees       |   < 5.6 km   |
+|      8      | 1/400 degrees        | 1/400 degrees      | < 280 meters |
+|     10      | 1/8000 degrees       | 1/8000 degrees     | < 14 meters  |
+|     11      | 1/40000 degrees      | 1/32000 degrees    |  < 4 meters  |
+|     12      | 1/200000 degrees     | 1/128000 degrees   |   < 90 cm    |
+|     13      | 1/1e6 degrees        | 1/512000 degrees   |   < 22 cm    |
+|     14      | 1/5e6 degrees        | 1/2.048e6 degrees  |    < 6 cm    |
+|     15      | 1/2.5e7 degrees      | 1/8.192e7 degrees  |   < 14 mm    |
+
+## Formatting
+
+A Plus Code consists of two or more **significant digits**, zero or more **padding characters** and exactly one **format separator**. Plus Code formatting has been selected to reduce writing errors and prevent spelling words.
+
+There are eight **digit places** to the left of the **format separator** and seven to the right:
+
+<kbd>1</kbd><kbd>2</kbd><kbd>3</kbd><kbd>4</kbd><kbd>5</kbd><kbd>6</kbd><kbd>7</kbd><kbd>8</kbd><kbd>+</kbd><kbd>9</kbd><kbd>10</kbd><kbd>11</kbd><kbd>12</kbd><kbd>13</kbd><kbd>14</kbd><kbd>15</kbd>
+
+Plus Codes SHALL NOT be formatted differently for right-to-left languages. This avoids ambiguity.
+
+A significant digit represents a numeric value as:
+
+| Value | 0    | 1    | 2    | 3    | 4    | 5    | 6    | 7    | 8    | 9    | 10   | 11   | 12   | 13   | 14   | 15   | 16   | 17   | 18   | 19   |
+| ----- | ---- | ---- | ---- | ---- | ---- | ---- | ---- | ---- | ---- | ---- | ---- | ---- | ---- | ---- | ---- | ---- | ---- | ---- | ---- | ---- |
+| Digit | 2    | 3    | 4    | 5    | 6    | 7    | 8    | 9    | C    | F    | G    | H    | J    | M    | P    | Q    | R    | V    | W    | X    |
+
+The padding character is defined as the zero ("0") character ([U+0030](http://unicode.org/charts/PDF/U0000.pdf)).
+
+The format separator is defined as the plus ("+") character ([U+002B](http://unicode.org/charts/PDF/U0000.pdf)).
+
+## Encoding
+
+A Plus Code uses significant digits to encode its southern and western bounds. The significant digits of a Plus Code represent latitude (north) and then longitude (east) offsets from the starting location 90° S, 180° W. 
+
+A significant digit in digit places 1, 3, 5, 7 and 9 represents an offset north equal to the north-south distance for that code length (see the next higher code length in the code length table above) multiplied by the significant digit value.
```

Comment:
Why "next higher" code length? If I have code length 4, I multiply the values in digit places 3 and 4 by 1°, not by (the "next higher" value of) 20°. If this is supposed to convey looking up (3+1)=4 in the table, I suggest rephrasing this to be more clear.

Reply:
Fixed at a2b41682dda221e01d6d42507c7860bcdb8f0e38